writer.go

```diff
@@ -16,7 +16,9 @@ type SafetyOpts struct {
 	EscapeCharMinus   bool
 	EscapeCharAt      bool
 	EscapeCharTab     bool
-	EscapeCharCR      bool
+	EscapeCharCR      bool // Escapes 0x0D (Carriage Return, '\r')
+	EscapeCharLF      bool // Escapes 0x0A (Line Feed, '\n')
+	OWASPSanitize     bool // Prepend single quote to dangerous fields only
```

Comment:
```suggestion
	PrependSingleQuote     bool // Prepend single quote to dangerous fields only (recommended OWASP sanitization)
```

I would rename it to "PrependSingleQuote" instead of OWASPSanitize, because other parameters are named to explain what they does. (feel free to suggest something different).

But the `OWASPSafe` default value is cool 😎